obal/data/roles/copr_repo/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: 'Create copr scratch build repo'
+  command: >
+    copr-cli
+      create
```

Comment:
will this command fail out if the copr repo is already created? I.E. when you are building a release to an existing copr repo?

Reply:
Ah, nevermind.  That 'when: build_package_scratch|default(false)' at the end of the whole block prevents it from running in the non-scratch case